examples/react/json-based-notification-renderer-with-css/src/components/hooks/utils/deleteNotificationFromCache.js

```diff
@@ -0,0 +1,40 @@
+import { GET_NOTIFICATIONS_QUERY } from '../useFetchNotifications'
+
+const deleteNotificationFromCache = (cache, mutationResult) => {
+  const queryOptions = {
+    query: GET_NOTIFICATIONS_QUERY,
+    variables: {
+      spaceId: process.env.REACT_APP_SPACE_ID,
+      input: {
+        location: 'TOKNOW',
+        filter: {
+          orderBy: 'DATE',
+        },
+      },
+    },
+  }
+
+  // Query the cache directly to get existing notifications
+  const cachedData = cache.readQuery(queryOptions)
+
+  // Grab id of notification that was deleted
+  const deletedId = mutationResult?.data?.deleteNotification || mutationResult?.data?.actionNotification
```

Comment:
Does this need updated based on the new changes to return the id?

Reply:
Since the mutations return the id directly and not within an object, this checks out fine